packages/sync/src/modules/Terms.php

```diff
@@ -11,6 +11,14 @@ function name() {
 		return 'terms';
 	}
 
+	public function get_object_by_id( $object_type, $id ) {
+		if ( $object_type === 'term' && $term = get_term( intval( $id ) ) ) {
```

Comment:
I wonder, can the `id` ever be non-integer? Isn't it always a `WP_Term` with an integer ID?

Reply:
I think it will always be an id.